hoss/coordinate_utilities.py

```diff
@@ -70,42 +75,56 @@ def any_absent_dimension_variables(varinfo: VarInfoFromDmr, variable: str) -> bo
     that have been created by opendap, but are not really
     dimension variables
     """
+
     return any(
         varinfo.get_variable(dimension) is None
         for dimension in varinfo.get_variable(variable).dimensions
     )
 
 
-def get_dimension_array_names_from_coordinate_variables(
+def get_dimension_array_names(
     varinfo: VarInfoFromDmr,
     variable_name: str,
 ) -> list[str]:
     """
-    Returns the dimensions names from coordinate variables
+    Returns the dimensions names from coordinate variables or from
+    configuration
     """
+    variable = varinfo.get_variable(variable_name)
+    dimension_names = varinfo.get_variable(variable_name).dimensions
+
+    if len(dimension_names) >= 2:
+        # if the dimension is not a variable, was added as a
+        # configuration entry,
+        if variable is not None:
+            return dimension_names
```

Comment:
The location here for `if variable is not none` is a bit odd.  I think this test could go immediately after getting the variable on line 93.

Also a comment below this if block indicating `# getting dimensions from coordinates` might be nice (nit)